doc/whats_new/v0.24.rst

```diff
@@ -445,6 +445,12 @@ Changelog
   of the hyperparameters are too tight.
   :issue:`12638` by :user:`Sylvain Lannuzel <SylvainLan>`.
 
+- |Fix| :class:`gaussian_process.GaussianProcessRegressor` raises division by
+  zero standard when normalization is requested
+  :pr:`19703` by :user:`<sobkevich>`,
+  :user:`Boris Villazón-Terrazas <boricles>` and
+  :user:`Alexandr Fonari <afonari>`.
```

Comment:
Hi @afonari, this should be moved in the 0.24.2 section. Thanks!

Reply:
Sorry, fixed.